build.sbt

```diff
@@ -48,6 +48,8 @@ lazy val scoverageSettings: Seq[Setting[_]] = Seq(
     "test\\..*",
     ".*(BuildInfo|Routes|Options).*",
     "logger.*\\(.*\\)",
+    ".*.controllers.amendment.group\\..*",
```

Comment:
do we have a ticket/todo to remind us to undo this?

Reply:
Good call. I'll add a TODO in there.

Reply:
Actually, I'm going to split these up into individual exclusions so that they can be reintroduced piecemeal.

Reply:
nice touch! will make that much easier :)